src/common/serializers/event.serializer.ts

```diff
@@ -58,7 +58,7 @@ export const deserializeEvent = (event: EventResponse): Event => {
         data: {
           directoryId: event.data.directory_id,
           user: deserializeDirectoryUser(event.data.user),
-          group: event.data.group,
+          group: deserializeEventDirectoryGroup(event.data.group),
```

Comment:
Actually looks like `EventDirectoryGroup` and `DirectoryGroup` are now identical. Could we make that full migration and also make this `deserializeDirectoryGroup`?

Sorry missed this before

Reply:
✅